src/maybe.ts

```diff
@@ -1,3 +1,7 @@
+import * as fl from './fantasy-land';
+
+export const binder = <T extends Function>(context: any, f: T): T => f.bind(context); // tslint:disable-line ban-types
```

Comment:
I'm working on adding something like this to the `impl` section of `SimplyTyped`. The stopper there is the fact that `bind`'s signature is `bind(context: any, ...args: any[])` so removing the first `n` args of the returned function will be difficult.

Anyways, this just makes it easier to _not_ have to respecify the types all the time.